src/graphql/queries/leaderboard.js

```diff
@@ -0,0 +1,53 @@
+const { isArray, each } = require('lodash');
+const { lowercaseFilters, runPaginatedQuery } = require('./utils');
+
+const buildFilters = ({
+  userAddress,
+  eventAddress,
+} = {}) => {
+  if (!eventAddress && !userAddress) {
+    throw Error('eventAddress or userAddress missing in filters');
+  }
+  const filters = [];
+  const filter = {};
+  if (eventAddress) filter.eventAddress = eventAddress;
+  if (userAddress) filter.userAddress = userAddress;
+
+  filters.push(filter);
+  return filters;
+};
+
+const eventLeaderboard = async (
+  root,
+  { filter, orderBy, limit, skip },
+  { db: { EventLeaderboard } },
+) => {
+  const query = filter ? { $or: buildFilters(lowercaseFilters(filter)) } : {};
```

Comment:
you need to add `userAddress` or use one of the fields that are already in there like `betterAddress` in the `lowercaseFilters` func